src/main/java/net/mcreator/ui/blockly/BlocklyEditorType.java

```diff
@@ -21,7 +21,8 @@
 
 public enum BlocklyEditorType {
 
-	PROCEDURE("procedures", "ptpl", "event_trigger"), AI_TASK("ai_setup", "aitpl", "aitasks_container");
+	PROCEDURE("procedures", "ptpl", "event_trigger"), AI_TASK("ai_setup", "aitpl", "aitasks_container"),
+	COMMAND_ARG("cmd_setup", "cmdtpl", "args_start");

```

Comment:
Maybe move these to separate lines (1 enum constant per line) and disable code formatter on them?

Reply:
It was Klemen's choice to make them on 1 line.

Reply:
Formatter does it like this